pkg/security/podsecuritypolicy/provider_test.go

```diff
@@ -482,8 +488,8 @@ func TestValidatePodSecurityContextSuccess(t *testing.T) {
 		Level: "level",
 	}
 
-	seccompNilWithNoAnnotations := defaultPod()
-	seccompNilWithNoAnnotationsPSP := defaultPSP()
+	seccompNilWithDefaultAnnotation := defaultPodWithSeccomp()
```

Comment:
The last round of feedback was that a default profile should be given if the PSP field was nil.  This means that when validating the annotation it should always have a value no matter what the PSP setting was.  These tests are going straight to validation without generation so we can test specific cases without worrying about PSP setup.  This requires the seccomp annotation to be set or it will give unexpected failures on cases where we're testing a non-related field.